shared/android/app/src/main/java/io/keybase/ossifrage/ChatBroadcastReceiver.java

```diff
@@ -0,0 +1,87 @@
+package io.keybase.ossifrage;
+
+import android.app.PendingIntent;
+import android.app.RemoteInput;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.RequiresApi;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+
+import keybase.Keybase;
+
+public class ChatBroadcastReceiver extends BroadcastReceiver {
+  public static String KEY_TEXT_REPLY = "key_text_reply";
+
+  @RequiresApi(api = Build.VERSION_CODES.KITKAT_WATCH)
+  private String getMessageText(Intent intent) {
+    Bundle remoteInput = RemoteInput.getResultsFromIntent(intent);
```

Comment:
As Nojima asked previously, is there any chance this can be intercepted by a different app?

Reply:
No, It uses an explicit intent (specifes what class to use for the intent) So it should stay within the application.
https://developer.android.com/training/articles/security-tips#use-intents 
> For security purposes, explicit intents are preferred.